### ISSUE TITLE
On the download of data sets

### ISSUE THREAD

**Issue by AI-Hello** [ivalab/AffKpNet#2]
An error occurs when the dataset is about to finish downloading

**Comment by ruinianxu** [comment]
> An error occurs when the dataset is about to finish downloading

Hi,

I just tested the download link and it works without any problem. You may consider download agian.

**Comment by AI-Hello** [comment]
Thank you，I've solved the problem, but I configure it according to your steps. The final version of enclib-cpu is 0.0.0. What's the problem？

**Comment by AI-Hello** [comment]
Can you send me the enclib-cup package you downloaded? Email: 1621540694@qq.com.Thank you!

**Comment by aragakiyui611** [comment]
I also faced this problem with dropbox, always failed at final 1%. May you upload these file on google drive? Thank you!

**Comment by AI-Hello** [comment]
> I also faced this problem with dropbox, always failed at final 1%. May you upload these file on google drive? Thank you!
yes,i have downloaded it. You need to download Dropbox software to download it normally.And I want to ask you，can you send me the enclib-cup package you downloaded? If enclib cup is configured successfully, can it be sent to me in the form of email:1621540694@qq.com.

**Comment by aragakiyui611** [comment]
> > I also faced this problem with dropbox, always failed at final 1%. May you upload these file on google drive? Thank you!
> > yes,i have downloaded it. You need to download Dropbox software to download it normally.And I want to ask you，can you send me the enclib-cup package you downloaded? If enclib cup is configured successfully, can it be sent to me in the form of email:[1621540694@qq.com](mailto:1621540694@qq.com).

Thank you, I will configure this later, haven't work on it yet.

**Comment by AI-Hello** [comment]
> > > 我也遇到过 dropbox 的这个问题，总是在最后 1% 时失败。你可以把这些文件上传到谷歌驱动器上吗？谢谢！
> > > 是的，我已经下载了。需要下载Dropbox软件才能正常下载。我想问你，你能把你下载的enclib-cup包发给我吗？如果enclib cup配置成功，能否以email的形式发给我：[1621540694@qq.com](mailto:1621540694@qq.com)。
> 
> 谢谢，我稍后会配置这个，还没有工作。

ok,thank you very much

**Comment by ruinianxu** [comment]
@aragakiyui611 Link of google drive has been updated in the readme.

**Comment by ruinianxu** [comment]
@AI-Hello Sorry for the late reply. I will look into it and reply to you as soon as possible.

**Comment by AI-Hello** [comment]
> @AI-Hello Sorry for the late reply. I will look into it and reply to you as soon as possible.

Thank you！

**Comment by ruinianxu** [comment]
@AI-Hello 
I looked into the problem but found that I never met this problem before. The package encoding/lib in the repo is what I used. I sent another one to your email and hope it works. If it doesn't, you may consider to see if there is any similar issues in [Pytorch-Encoding](https://github.com/huanghoujing/PyTorch-Encoding) or [DANet](https://github.com/junfu1115/DANet).

**Comment by AI-Hello** [comment]
> @AI-Hello
> I looked into the problem but found that I never met this problem before. The package encoding/lib in the repo is what I used. I sent another one to your email and hope it works. If it doesn't, you may consider to see if there is any similar issues in [Pytorch-Encoding](https://github.com/huanghoujing/PyTorch-Encoding) or [DANet](https://github.com/junfu1115/DANet).

ok, Thank you very much！

**Comment by AI-Hello** [comment]
> @AI-Hello
> 我调查了这个问题，但发现我以前从未遇到过这个问题。回购中的包 encoding/lib 是我使用的。我向您的电子邮件发送了另一个，希望它有效。如果没有，您可以考虑查看[Pytorch-Encoding](https://github.com/huanghoujing/PyTorch-Encoding)或[DANet 中](https://github.com/junfu1115/DANet)是否存在类似问题。

Hello, the previous error has been resolved, but the data set you gave is missing train_umdself_kp_cateSplit.txt and val_umdself_kp_cateSplit.txt and test_umdself_kp_cateSplit.txt files

**Comment by ruinianxu** [comment]
@AI-Hello 
Actually you don't need two additional split files for train and val sets. If you look at the split file provided, 1 represents training and 0 represents validation. Sorry for the error. I will update the code later today.

**Comment by ruinianxu** [comment]
@AI-Hello 
I've updated the readme about how to download dataset. Except two txt files which record path for ground-truth of image, mask and keypoint files, you will also need another folder which stores all mask files. I've also uploaded these mask files to both dropbox and google drive. Please make sure the name of your folder is correct.

**Comment by AI-Hello** [comment]
> @AI-Hello
> I've updated the readme about how to download dataset. Except two txt files which record path for ground-truth of image, mask and keypoint files, you will also need another folder which stores all mask files. I've also uploaded these mask files to both dropbox and google drive. Please make sure the name of your folder is correct.

Thank you, but Google needs to apply for permission to download the data set you gave

**Comment by AI-Hello** [comment]
> @AI-Hello
> I've updated the readme about how to download dataset. Except two txt files which record path for ground-truth of image, mask and keypoint files, you will also need another folder which stores all mask files. I've also uploaded these mask files to both dropbox and google drive. Please make sure the name of your folder is correct.

Thank you, but Google needs to apply for permission to download the data set you gave. But I didn’t find the files you mentioned, do I need to download the entire dataset file again?

**Comment by AI-Hello** [comment]
> @AI-Hello
> I've updated the readme about how to download dataset. Except two txt files which record path for ground-truth of image, mask and keypoint files, you will also need another folder which stores all mask files. I've also uploaded these mask files to both dropbox and google drive. Please make sure the name of your folder is correct.

I also think we can add a QQ or WeChat to discuss together, because I am also working in this direction.

**Comment by ruinianxu** [comment]
@AI-Hello Updated. And you might also need extra matlab files for evaluation. I will update it as soon as possible.

**Comment by ruinianxu** [comment]
Instruction of evaluating affordance segmentation mask has been updated. Please let me know if there is any issues.

**Comment by AI-Hello** [comment]
Download split files for training and validation sets which are named train_umdself_kp_cateSplit.txt and test_umdself_kp_cateSplit.txt.  In this sentence, I didn't find these two txt files in the dataset, and there is no dataset in the google drive you gave me.Can you put two txt files on Google drive or UMD+GT alone？Thank you

**Comment by ruinianxu** [comment]
@AI-Hello I am sorry that I don't know why the link didn't work before. I've updated everything in the google drive. Hope it hopes this time.

**Comment by AI-Hello** [comment]
Hello,your program has a lot of bugs, and the storage location of the data set has been wrong
